tool/tctl/common/kube_command.go

```diff
@@ -83,3 +84,29 @@ func (c *KubeCommand) ListKube(ctx context.Context, client auth.ClientI) error {
 		return trace.BadParameter("unknown format %q", c.format)
 	}
 }
+
+var kubeMessageTemplate = template.Must(template.New("kube").Parse(`The invite token: {{.token}}
+This token will expire in {{.minutes}} minutes.
+
+To use with Helm installation follow these steps:
+
+# Retrieve the Teleport helm charts
+helm repo add teleport https://charts.releases.teleport.dev
+# Refresh the helm charts
+helm repo update
+
+> helm install teleport-agent teleport/teleport-kube-agent \
+  --set kubeClusterName=cluster ` + "`" + `# Change kubeClusterName variable to your preferred name.` + "`" + ` \
+  --set proxyAddr={{.auth_server}} \
+  --set authToken={{.token}} \
+  --create-namespace \
+  --namespace=teleport-agent
+        
+Please note:
+        
+  - This invitation token will expire in {{.minutes}} minutes.
+  - {{.auth_server}} must be reachable from Kubernetes cluster.
+  - The token is usable in a standalone Linux server with kubernetes_service.
+  - See goteleport.com/docs/kubernetes-access/ for detailed installation information.
```

Comment:
In the desktop template the docs link includes https:// but that's missing here.

Reply:
thanks for reviewing @zmb3 this part, fixed.